projects/mci/src/regen_braking.c

```diff
@@ -0,0 +1,27 @@
+#include "regen_braking.h"
+
+#include "can.h"
+#include "can_msg_defs.h"
+#include "can_unpack.h"
+#include "status.h"
+
+// regen braking state 1:ON - 0:OFF
```

Comment:
Probably would be good just to define the states somewhere with an enum

Reply:
done